package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>ssdf_msgs</name>
+  <version>0.0.0</version>
+  <description>The ssdf_msgs package</description>
+
+  <author email="vinhloiit1327@gmail.com">Loi Ly</author>
+  <maintainer email="vinhloiit1327@gmail.com">Loi Ly</maintainer>
```

Comment:
You would not public your personal information in open source project

Reply:
Okay I will update these fields